### PR TITLE
Modify for redesigned stitching

### PR DESCRIPTION
**Pull request by sebatyler**
## Changes
- Jira Ticket: [DATACENTER-5252]
- Define a new docker image for the redesigned stitching program
- https://www.notion.so/bepro/bepro-video-pipeline-06e77f42f8804699b90988ac64c5747f

[DATACENTER-5252]: https://beprocompany.atlassian.net/browse/DATACENTER-5252?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: stitching_base/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,40 @@
-FROM bepro/tensorflow_cc:ubuntu-cuda-2.5.0
+FROM floopcz/tensorflow_cc:ubuntu-cuda-2.6.0
```
Comment (sebatyler) [inline stitching_base/Dockerfile:1]: The recent version includes the 8.6 capability which is for RTX 3080 as the default value.
https://github.com/FloopCZ/tensorflow_cc/commit/bcf01eb552ce79a3dfc83a417a6f6858cc811f14
https://hub.docker.com/layers/floopcz/tensorflow_cc/ubuntu-cuda-2.6.0/images/sha256-8d61195134bed90ed1568c0a2882b6f8f8e0a9fae4546586c2453ff545a3c865?context=explore

So we don't need to use our own docker image any more.
https://github.com/bepro-company/tensorflow_cc/pull/1/files

https://developer.nvidia.com/cuda-gpus
image

**Review by bdwoo — APPROVED** [review]
확인했습니다.

**Review by bdwoo — APPROVED** [review]

**Comment by sebatyler** [comment]
확인 감사합니다.
build.sh 에 install path 추가되어 테스트 가능해지면 머지하도록 하겠습니다.

**Comment by sebatyler** [comment]
CMakeLists.txt 에서 spdlog 설치하는 것 뺀건 테스트 못해봤지만 설치하는 건 정의했고 -p 옵션도 잘 되는 것까지 확인해서 머지하도록 하겠습니다.

**Pull request merged**